packages/test/src/workflows/local-activity-testers.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Workflows used by `test-local-activities.ts`
+ * @module
+ */
+
+import * as wf from '@temporalio/workflow';
+import type * as activities from '../activities';
+
+const { echo, throwAnError, waitForCancellation } = wf.proxyLocalActivities<typeof activities>({
+  startToCloseTimeout: '1m',
+});
+
+export async function runOneLocalActivity(s: string): Promise<string> {
+  return await echo(s);
+}
+
+export async function runParallelLocalActivities(...ss: string[]): Promise<string[]> {
+  return await Promise.all(ss.map(echo));
+}
+
+export async function throwAnErrorFromLocalActivity(message: string): Promise<void> {
+  await throwAnError(true, message);
+}
+
+export async function throwAnExplicitNonRetryableErrorFromLocalActivity(message: string): Promise<void> {
+  const { throwAnError } = wf.proxyLocalActivities<typeof activities>({
+    startToCloseTimeout: '1m',
+    retry: { nonRetryableErrorTypes: ['Error'] },
+  });
+
+  await throwAnError(false, message);
+}
+
+export async function throwARetryableErrorWithASingleRetry(message: string): Promise<void> {
+  const { throwAnError } = wf.proxyLocalActivities<typeof activities>({
+    startToCloseTimeout: '1m',
+    retry: { maximumAttempts: 2 },
+  });
+
+  await throwAnError(false, message);
+}
+
+export async function cancelALocalActivity(): Promise<void> {
+  await wf.CancellationScope.cancellable(async () => {
+    const p = waitForCancellation();
+    await wf.sleep(1);
+    wf.CancellationScope.current().cancel();
+    await p;
+  });
+}
+
+export async function runSerialLocalActivities(): Promise<void> {
+  await echo('1');
+  await echo('2');
+  await echo('3');
+}
+
+export async function throwAnErrorWithBackoff(): Promise<void> {
+  const { succeedAfterFirstAttempt } = wf.proxyLocalActivities({
+    startToCloseTimeout: '1m',
+    localRetryThreshold: '1s',
+    retry: { maximumAttempts: 2, initialInterval: '2s' },
+  });
+
+  await succeedAfterFirstAttempt();
+}
+
+export async function runANonExisitingLocalActivity(): Promise<void> {
+  const { activityNotFound } = wf.proxyLocalActivities({
+    startToCloseTimeout: '1m',
+    retry: { maximumAttempts: 1 },
+  });
+
+  await activityNotFound();
+}
+
+export const interceptors: wf.WorkflowInterceptorsFactory = () => {
+  return {
+    outbound: [
+      {
+        async startTimer(input, next) {
+          const { test } = wf.proxySinks();
+          await next(input);
```

Comment:
When this resolves, start timer request has been received by core or Server, or timer has gone off / completed? I think former, but that feels different from the `next`s in `scheduleActivity`, so I'll document it.

Reply:
Timer has been completed when this resolves.
The name is of this method is misleading.